src/blog/lagom-1-4-7.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Lagom 1.4.7 released!
+date: 2018-07-19
+author_github: marcospereira
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.4.7
+---
+
+The Lagom team has released Lagom 1.4.7.
+
+This release provides several bug fixes and updates.
+
+## Highlights
+
+- Updates Akka to [2.5.14](https://akka.io/blog/news/2018/07/13/akka-2.5.14-released)
+- Updates Akka to [2.6.17](https://blog.playframework.com/play-2-6-17-released/). Play 2.6.17 supports starting the application even when a database is not available, but the default configuration is to fail fast. In Lagom, this configuration was overridden to not fail fast.
```

Comment:
`s/Updates Akka/Updates Play`

This line in particular only mentions Play 2.6.17 but I think the gap since Lagom `1.4.6` includes also `2.6.15` and `2.6.16` so it'd be good to link to those release notes too.

"Play 2.6.17 supports starting the application even when a database is not available". Lagom defaults to cassandra which already supported that. It'd be good to specify the change applies to RDBMS.